engine/sound/src/sound.cpp

```diff
@@ -1267,6 +1266,7 @@ namespace dmSound
 
                 char* buffer = ((char*) instance->m_Frames) + instance->m_FrameCount * stride;
                 uint32_t buffer_size = n * stride;
+                uint32_t decoded = 0;
```

Comment:
The decoders has changed behavior when skipping (cumulative), so the calculated frame count became too large.
Since this is in a loop anyways, it was always a bug imho, that the `decoder` wasn't set to 0.